src/bms/parse/random.rs

```diff
@@ -1,13 +1,16 @@
+//! Random Part.
+
 mod ast_build;
 mod ast_parse;
+pub mod rng;
 
 use thiserror::Error;
 
-use super::{ParseWarning, rng::Rng};
+use super::ParseWarning;
 use crate::bms::{lex::token::Token, parse::BmsParseTokenIter};
 
-use self::ast_build::*;
-use self::ast_parse::*;
+pub use self::rng::*;
```

Comment:
Please don't re-export wildcard imports. It makes that this module strongly depends on `rng` module's items. I want to avoid that.